demisto_sdk/tests/test_files/test_fake_playbook.yml

```diff
@@ -0,0 +1,179 @@
+id: test_fake_playbook
```

Comment:
please rename the playbook to something that makes it easy to tell where it's used, like `test_playbook_value_starting_with_equal_sign.yml`